src/org/opensolaris/opengrok/authorization/AuthorizationFramework.java

```diff
@@ -141,6 +151,38 @@ public void setPluginDirectory(String directory) {
         setPluginDirectory(directory != null ? new File(directory) : null);
     }
 
+    /**
+     * Make {@code reload()} search for plugins in class files.
+     * @param flag true or false
+     */
+    public void setLoadClasses(boolean flag) {
+        loadClasses = flag;
+    }
+    
+    /**
+     * Whether to search for plugins in class files.
+     * @return true if enabled, false otherwise
+     */
+    public boolean getLoadClasses() {
+        return loadClasses;
+    }
+    
+    /**
+     * Make {@code reload()} search for plugins in jar files.
+     * @param flag true or false
+     */
+    public void setLoadJars(boolean flag) {
+        loadJars = flag;
+    }
+    
+    /**
+     * Whether to search for plugins in class files.
+     * @return true if enabled, false otherwise
+     */
+    public boolean getLoadJars() {
+        return loadJars;
+    }
+
```

Comment:
and also `loadPluginsFiles` or simiar for both of these may be of use in the future

Reply:
if you mean a setter for both I am hesitant to add unused code.

Reply:
ok